hcn/hcnsupport.go

```diff
@@ -76,6 +106,18 @@ func GetSupportedFeatures() SupportedFeatures {
 	features.NetworkACL = isFeatureSupported(globals.Version, NetworkACLPolicyVersion)
 	features.NestedIpSet = isFeatureSupported(globals.Version, NestedIpSetVersion)
 
+	// Only print the HNS version and features supported once, instead of everytime this is invoked. These logs are useful to
```

Comment:
Should we be using the term HCN here instead of HNS? That seems to be the publicly documented terminology, and this is an exported package.